hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java

```diff
@@ -146,6 +146,7 @@ public static ContainerCommandResponseProto logAndReturnError(
     log.info("Operation: {} , Trace ID: {} , Message: {} , Result: {}",
         request.getCmdType().name(), request.getTraceID(),
         ex.getMessage(), ex.getResult().getValueDescriptor().getName());
+    log.error("StorageContainerException Occurred!", ex);
```

Comment:
We already have a log.info just above it with exception message and the result. If more information about the exception is needed, can we consolidate it into the previous message.

Reply:
Thanks @xiaoyuyao for the comment.
Updated.